src/apiProxy/apiProxyRoutes.js

```diff
@@ -11,13 +7,7 @@ const validApiPayloadSchema = Joi.object({
 	logout: Joi.any(),
 });
 
-const getApiProxyRoutes = (path, env, apiProxyFn$) => {
-	const proxyApiRequest$ = apiProxyFn$({
-		API_TIMEOUT: env.API_TIMEOUT,
-		baseUrl: env.API_SERVER_ROOT_URL,
-		duotoneUrls: getDuotoneUrls(duotones, env.PHOTO_SCALER_SALT),
-	});
```

Comment:
these config values are now read directly from the server config in `server.app`